mapApp/templates/mapApp/overlays.html

```diff
@@ -14,6 +14,9 @@
     {% trans "Cancel" %}
   </button>
 </div>
+<!-- <div id="pointCounterContainer" class="leaflet-container leaflet-bottom leaflet-left">
```

Comment:
Would you mind adding a comment in explaining what this does and why it's commented out (for future reference by others)? Thanks!